applications/GeoMechanicsApplication/custom_elements/small_strain_U_Pw_diff_order_element.cpp

```diff
@@ -1915,7 +1895,7 @@ GeometryData::IntegrationMethod SmallStrainUPwDiffOrderElement::GetIntegrationMe
 
 SizeType SmallStrainUPwDiffOrderElement::GetVoigtSize() const
```

Comment:
Since this wrapper function has no additional value anymore, I would suggest to eliminate it completely. (It used to have logic inside of it, but not any longer.)

Reply:
removed.